clpy/backend/opencl/env.pyx

```diff
@@ -18,6 +21,40 @@ num_platforms = api.GetPlatformIDs(1, &__platforms_ptr[0])
 cdef cl_platform_id primary_platform = __platforms_ptr[0]
 logging.info("SUCCESS")
 
+
+# Platform Version Check
+cdef size_t param_value_size
```

Comment:
This variable doesn't have to be in global space. Can you move them into a function?